src/layouts/header.html

```diff
@@ -1,22 +1,20 @@
 <header>
-<div class="header-container">
-   
-    
-    <figure><img class="icon_align--right" src="../assets/icons/align-right.svg" alt="Menu_Icon"></figure>
-    
-    <a href="#" class="home_icon--left">Vaccination.ng</a>
+    <div class="header-container">
+        <figure><img class="icon_align--right" src="../assets/icons/align-right.svg" alt="Menu_Icon"></figure>
 
-    <nav class="nav">
-        <ul>
-            <include src="./components/navList.html" locals='{
+        <a href="#" class="home_icon--left">Vaccination.ng</a>
+
+        <nav class="nav">
+            <ul>
+                <include src="./components/navList.html" locals='{
```

Comment:
I don't see this file in ur src. where is it?

Reply:
It's still here
![image](https://github.com/Hoa0304/practice-html-css/assets/117150692/fd048c3f-cb4f-4383-99d8-930f41060693)